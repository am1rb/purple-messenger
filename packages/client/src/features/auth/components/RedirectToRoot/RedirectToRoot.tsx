import React from "react";
import { Redirect } from "react-router";

function RedirectToRoot() {
  return <Redirect to="/" />;
}

export default RedirectToRoot;
