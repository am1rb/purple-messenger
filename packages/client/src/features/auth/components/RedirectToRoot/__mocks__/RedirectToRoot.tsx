import React from "react";

function RedirectToRoot() {
  return <span data-testid="mock-redirect-to-root" />;
}

export default RedirectToRoot;
