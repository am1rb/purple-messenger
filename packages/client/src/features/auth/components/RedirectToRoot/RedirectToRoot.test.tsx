import React from "react";
import { render } from "@testing-library/react";
import RedirectToRoot from "./RedirectToRoot";

describe("The <RedirectToRoot /> tests", () => {
  it("Should match the snapshot", () => {
    const { container } = render(<RedirectToRoot />);
    expect(container).toMatchSnapshot();
  });
});
