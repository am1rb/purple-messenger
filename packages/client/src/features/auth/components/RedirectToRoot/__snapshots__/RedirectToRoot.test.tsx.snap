// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The <RedirectToRoot /> tests Should match the snapshot 1`] = `
<div>
  <span
    data-testid="mock-redirect"
    data-to="/"
  />
</div>
`;
